src/electron/flux/types/electron-window-state-store-data.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export type electronRoutes = 'deviceConnectView' | 'resultsView';
```

Comment:
make this caps. ElectronRoutes

Reply:
not sure if using `electron` here is a good idea (same reason as in my comment on `src/common/stores/store-names.ts`)

Reply:
renamed to viewroutes in window-state-store-data